install/streams.csv

```diff
@@ -1654,10 +1654,11 @@ QRZ,10,50,false,false,true,1990-04-26T04:39:00Z,2002-02-18T05:58:00Z
 QRZ,10,100,false,true,false,2003-08-06T00:00:00Z,2009-08-12T23:00:00Z
 QRZ,10,100,false,false,false,2009-08-12T23:01:01Z,9999-01-01T00:00:00Z
 QRZ,20,1,false,false,false,2003-08-06T00:00:00Z,2015-11-12T01:00:00Z
-QRZ,20,50,false,false,false,2009-08-12T23:01:01Z,2019-11-21T22:55:00Z
-QRZ,20,200,false,false,true,2003-08-06T00:00:00Z,2019-11-21T22:55:00Z
-QRZ,21,50,false,false,false,2019-11-21T22:56:00Z,2020-06-16T00:00:00Z
-QRZ,21,200,false,false,true,2019-11-21T22:56:00Z,9999-01-01T00:00:00Z
+QRZ,20,50,false,false,false,2009-08-12T23:01:01Z,2015-11-12T01:00:00Z
+QRZ,20,200,false,false,true,2003-08-06T00:00:00Z,2015-11-12T01:00:00Z
+QRZ,21,50,false,false,false,2015-11-12T02:00:00Z,2020-06-16T00:00:00Z
+QRZ,21,200,false,false,true,2015-11-12T01:00:00Z,2020-02-08T23:59:59Z
```

Comment:
- should LC 21 at 200hz start at `2015-11-12T02:00:00Z`?
- how do we know that the LC 21 200hz triggered ends in` 2020-02-08T23:59:59Z`?

Reply:
All good for the second question on ending of triggered 200 and start of continuous 200:

```
(base) murieln@ubuntu:~$ aws s3 ls --profile sod_readrole s3://geonet-archive/miniseed/2020/2020.040/QRZ.NZ/
2020-02-14 21:21:09    3149824 2020.040.QRZ.21-BNE.NZ.D
2020-02-14 21:21:08    3107840 2020.040.QRZ.21-BNN.NZ.D
2020-02-14 21:21:08    3238400 2020.040.QRZ.21-BNZ.NZ.D
2020-02-14 21:21:07    1886208 2020.040.QRZ.21-HNE.NZ.D
2020-02-14 21:21:07    1928192 2020.040.QRZ.21-HNN.NZ.D
2020-02-14 21:21:07    2078720 2020.040.QRZ.21-HNZ.NZ.D

```

Reply:
have amended the dates to something that matches the data set